test/lib.sh

```diff
@@ -26,7 +26,7 @@ add_cleanup() {
 }
 
 empty_repo() {
-    tmpd=$(mktemp -d)
+    tmpd=$(mktemp -d -t boilerplate_test_XXXXXXXX)
```

Comment:
micronit: any reason underscores here and hyphens [below](https://github.com/openshift/boilerplate/pull/17/files/a45b4cf538a8ad3837bbaff5b5f04b1264d3641b..b6d0ef4e5e6853ce4322b9f111268f688a0abd8c#diff-e1b2b21859e872e9af54745ed0a1b5c4R163)?

Reply:
nope, fixed